ci/scripts/build-java-lambdas.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/ash
+# shellcheck shell=dash
+set -euo pipefail
+
+cd src || exit
```

Comment:
This should be `exit 1` so it exits with an error. Otherwise if the src directory doesn't exist the job will still complete because this exit exit's with success. But actually because you have `set -e` you can get rid of the || exit ecompletely since if the cd fails the script will exit as an error:

```suggestion
cd src
```